README.md

```diff
@@ -21,4 +21,4 @@ Aspect | Information
 ![Michael Jackson](Michael_Jackson_-_Man_in_the_Mirror.png)
 
 * You are welcome to add me on Facebook (~~although I don't talk a lot~~)! :point_down:
```

Comment:
Smart use of an emoji and great use of the link element!

Reply:
Hi Nadia! Great suggestion for an alternative way to embed an image. Although, the image was able to render for me, so I'm not sure why you can't see it.

Reply:
Overall Yan did a great work includding all the required elements beisdes the the minor bug with the image.